examples/sessions.rs

```diff
@@ -0,0 +1,37 @@
+use tide::prelude::*;
+use tide::sessions::SessionMiddleware;
```

Comment:
This is bigger but I think we may want to reorganize middleware into `tide::middleware` similar to Surf.